bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/TestCheckAndMutateUtil.java

```diff
@@ -150,12 +154,14 @@ public void testRowMutations() throws IOException {
 
     RowMutations rowMutations = new RowMutations(rowKey);
     rowMutations.add(PUT);
-    CheckAndMutateRowRequest result = underTest
+
+    ConditionalRowMutation conditionalRowMutation = underTest
```

Comment:
Please add `.toProto(REQUEST_CONTEXT)` in this chain instead of having two variables.